Ghiurutan V. Alexandru/Homework5/src/model/Word.java

```diff
@@ -0,0 +1,38 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Observable;
+
+public class Word extends Observable {
+
```

Comment:
Here I use the Observer and Composite design patterns.
Composite is used effectively when adding a new word to the dictionary and Observer is used especially at deletion.Both ensure consistency of the dictionary.